src/utils/components/DetailItem/DetailItemHeader.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, ReactNode } from 'react';
+
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  DescriptionListTerm,
+  DescriptionListTermHelpTextButton,
+  Popover,
+} from '@patternfly/react-core';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+
+import './DetailItem.scss';
+
+type DetailItemHeaderProps = {
+  bodyContent: ReactNode;
+  breadcrumb?: string;
+  descriptionHeader: ReactNode;
+  isPopover: boolean;
+  label?: ReactNode;
+  maxWidth?: string;
+  moreInfoURL?: string;
+};
+
+export const DetailItemHeader: FC<DetailItemHeaderProps> = ({
```

Comment:
why making such a genric component? cant split for 2 ?

Reply:
also copied from kubevirt